catalog/ui/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -25,11 +25,12 @@ define([
     'component/property/property',
     'component/dropdown/dropdown',
     'component/dropdown/dropdown.view',
+    './filter-param',    
```

Comment:
from a file structure standpoint, is top level consistent with other files?  Or is this more of a component?  I see it is in the same location as filter.hbs... so I am a little confused about the structure and this is just an inquiry, not a recommendation.